### PR TITLE
Update schemas for ASDF standard 1.6.0

### PR DESCRIPTION
**Pull request by braingram**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -2,20 +2,21 @@
...
     'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
     'Development Status :: 5 - Production/Stable',
 ]
 dependencies = [
-    'asdf-standard >= 1.0.1',
-    'importlib_resources >= 3; python_version<"3.9"',
+    #'asdf-standard >= 1.0.1',
+    'asdf-standard @ git+https://github.com/braingram/asdf-standard.git@split_core'
```
Comment (braingram) [inline pyproject.toml:19]: To be removed prior to merge and after https://github.com/asdf-format/asdf-standard/pull/422 is merged and released.

File: pyproject.toml [hunk 2]
```diff
@@ -33,7 +34,8 @@ docs = [
 ]
 test = [
     'asdf >= 2.8.0',
-    'asdf-astropy',
+    #'asdf-astropy',
+    'asdf-astropy @ git+https://github.com/braingram/asdf-astropy.git@split_core',
```
Comment (braingram) [inline pyproject.toml:38]: To be removed prior to merge. This is at the moment required due to examples using tags new to this PR which are not yet supported in asdf-astropy (support added in: https://github.com/astropy/asdf-astropy/pull/219).

**Review by nden — APPROVED** [review]
I am not sure why there's one failure with remap-axes-1.4.0. 
LGTM

**Comment by braingram** [comment]
Thanks for reviewing this PR.

The failure of the `CI/Run schemas and dev extensions` job is due to `asdf-astropy` being installed from main (instead of the branch specified in the `pyproject.toml` test section in this PR). The main version isn't aware of the new manifest added in this PR (so it's not aware of `remap_axis-1.4.0`). It stems from the "chicken egg" that is this package and `asdf-astropy` (where a change here is required before `asdf-astropy` yet that same change "breaks" `asdf-astropy` as seen here).

**Comment by braingram** [comment]
As this PR no longer adds asdf-astropy-dev as a dependency in `pyproject.toml` the failures mentioned above for `Run schemas and dev extensions` now appear in other CI runs (like `CI / Python 3.11 Schema validation tests`). Once this PR is merged and this package released asdf-astropy can be updated to fix the CI.

**Pull request merged**